src/Microsoft.Health.Fhir.Liquid.Converter/DotLiquids/TemplateLocalFileSystem.cs

```diff
@@ -33,7 +33,7 @@ public TemplateLocalFileSystem(string templateDirectory, DataType dataType)
 
         public string ReadTemplateFile(Context context, string templateName)
         {
-            throw new NotImplementedException();
+            return ReadTemplateFile(templateName);
```

Comment:
This is a very good question. I also tried to just make the contextless override public but something wasn't picking it up. I assume I missed an interface somewhere. I'm hoping Justin can help me with this `CodeMapping` stuff because it's a mess.

Reply:
Ahhhhh okay, if we want ReadTemplateFile to be publix it has to accept that `Context` param to conform to a interface deep in DotLiquid.... I actually think what I have here is a decent solution now.